vignettes/measures-vignette.Rmd

```diff
@@ -0,0 +1,166 @@
+---
+title: "measures-vignette"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{measures-vignette}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(mlr3fairness)
+library(mlr3verse)
+library(mlr3)
+library(ranger)
+```
+# Why should we measure fairness?
+
+In machine learning, the problem of algorithmic bias and fairness problems are well known and well studied. Outcomes may be skewed by a range of factors and thus might be considered unfair to certain groups or individuals. An example would be the way social media sites deliver personalized news to consumers. The potential is always there to give undue weight to certain viewpoints. Thus, we want to deliver a fairness toolkit for mlr3 users so they could measure the fairness metrics when building the models. And moreover, correcting the algorithmic bias intends to ensure fairness for protected fields. 
+
+# Fairness Measures Demonstration
+
+In this vignette, users could see examples on 
+
+* How to create the most common fairness measures.
+* How to customize their fairness measures using MeasureFairness classes.
+* How to compare different learners fairness levels using Benchmarks.
+
+```{r}
+#Load the dataset, check the protected column has been set and make the predictions.
+adult_train = tsk("adult_train")
+adult_test = tsk("adult_test")
+adult_train$col_roles$pta
+
+learner_obj = lrn("classif.rpart")
+learner_obj$train(adult_train)
+predictions = learner_obj$predict(adult_test)
+```
+
```

Comment:
Structure: Start with motivation, then go into how to use the pre-defined measures  (via msr) and THEN show how to create your own measure.

Reply:
Got it, reorganize the vignettes.